vendor/bat-native-ads/src/bat/ads/internal/ad_server/get_catalog_url_request_builder.cc

```diff
@@ -27,6 +27,7 @@ UrlRequestPtr GetCatalogUrlRequestBuilder::Build() {
 ///////////////////////////////////////////////////////////////////////////////
 
 std::string GetCatalogUrlRequestBuilder::BuildUrl() const {
+  // TODO(Moritz Haller): Bump catalog version
```

Comment:
Discussed with @moritzhaller and this change will be made once the server side is ready. There are a few places that need changing.